app/main/views/services.py

```diff
@@ -41,6 +43,10 @@ def list_services():
 
     services = Service.query.filter(
         Service.framework.has(Framework.expired == false())
+    ).order_by(
+        asc(Service.framework_id),
```

Comment:
Migrations are ordered and even though we do have occasional issues applying them, I don't think we can end up with different order on different environments, especially since we deploy the same version to all the stages. So I'm not sure what's the concern here.

ID order is more arbitrary than name order would be, but unless we have another preference, I don't think it's a problem.